independent-publisher-connectors/TSheets Quickbooks Time/apiDefinition.swagger.json

```diff
@@ -0,0 +1,654 @@
+                        "x-ms-summary": "Modified Before",
+                        "description": "in ISO 8601 format (YYYY-MM-DDThh:mm:ss\u00b1hh:mm)",
+                        "x-ms-visibility": "advanced"
+                    },
+                    {
+                        "name": "modified_since",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "x-ms-summary": "Modified Since",
+                        "description": "in ISO 8601 format (YYYY-MM-DDThh:mm:ss\u00b1hh:mm).",
+                        "x-ms-visibility": "advanced"
+                    },
+                    {
+                        "name": "supplemental_data",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "x-ms-summary": "Supplemental Data",
+                        "enum": [
+                            "yes",
+                            "no"
+                        ],
+                        "description": "Indicates whether supplemental data should be returned.",
+                        "x-ms-visibility": "advanced",
+                        "default": "yes"
+                    },
+                    {
+                        "name": "per_page",
+                        "in": "query",
+                        "required": false,
+                        "type": "integer",
+                        "x-ms-summary": "Per Page",
+                        "description": "Represents how many results you'd like to retrieve per request.",
+                        "x-ms-visibility": "advanced"
+                    },
+                    {
+                        "name": "page",
+                        "in": "query",
+                        "required": false,
+                        "type": "integer",
+                        "x-ms-summary": "Page",
+                        "description": "Represents the page of results you'd like to retrieve.",
+                        "x-ms-visibility": "advanced"
+                    },
+                    {
+                        "name": "active",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "x-ms-summary": "Active",
+                        "description": "'yes', 'no', or 'both'.",
+                        "enum": [
+                            "yes",
+                            "no",
+                            "both"
+                        ],
+                        "x-ms-visibility": "advanced",
+                        "default": "yes"
+                    }
+                ]
+            }
+        },
+        "/projects": {
+            "get": {
+                "responses": {
+                    "default": {
+                        "description": "default",
+                        "schema": {}
```

Comment:
@artesianst empty schema is not allowed please fix this in all your swagger file